apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -225,11 +225,18 @@ def _read_dataframe(
 
 
 # ----------------------------------------------------------------
+class Sentinel:
```

Comment:
Nit: this (and MISSING) are generally useful, so might consider putting them into a common utils file?

Reply:
Thanks again @bkmartinjr !